includes/class-coblocks-block-assets.php

```diff
@@ -548,6 +551,31 @@ public function frontend_scripts() {
 				'rightLabel' => __( 'Next', 'coblocks' ),
 			)
 		);
+
+		$header = $this->get_template_content( 'header' );
```

Comment:
This is pretty smart. I am in the process of moving blocks to register by PHP which has the added benefit of loading the script any time the block appears as opposed to this 'has_block' logic which doesn't account for enough situations.